### ISSUE TITLE
[Application] Confirm with STMAS that application form is ready 

### ISSUE THREAD

**Issue by michael-markl** [digitalfabrik/entitlementcard#629]
Once we have a first idea how the application form looks like (and the ping-pong with the organization is implemented in a first draft version), ask the STMAS for changes and feedback.

Blocked by #628 and #627 .

**Comment by dkehne** [comment]
- We need one more field "Adresszusatz" which is not manditory. In case of sending the PDF document printed as a letter, it might me needed. 
- telephone number shouldn't be mandatory. 
- label adjustment "Die Kontaktperson hat der Weitergabe seiner Daten zum Zwecke der Antragsverarbeitung zugestimmt." to "Die Kontaktperson hat der Weitergabe seiner Daten zum Zwecke der Antragsverarbeitung zugestimmt und darf zur Überprüfung kontaktiert werden."
- Angaben zur Organisation: Not mandatory: Land (Volunteering abroad is also valid i guess)
- Nice to have: Tätig seit *: Can this date picker easiley be adjusted to month+year only instead of exact day?
- label adjustment: "Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. Die Datei darf maximal 5 MB groß sein und muss im JPG, PNG oder PDF Format sein." to "Falls vorhanden: hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. Die Datei darf maximal 5 MB groß sein und muss im JPG, PNG oder PDF Format sein."
- label adjustment: "Ihr Antrag für die Ehrenamtskarte wurde erfolgreich übermittelt. Sie können das Fenster schließen." to "Über den Fortschritt Ihres Antrags werden Sie per E-Mail informiert. Sie können das Fenster jetzt schließen.". If possible technically, add sth like "Für Rückfragen zum Vorgang notieren Sie sich bitte folgende Bearbeitungs-ID [REGIONID]-XXXX"

**Comment by maxammann** [comment]
Awesome, after that we can count the form as being ready?

**Comment by dkehne** [comment]
This additional feedback came also in, so should be considered as well: 
Card type:

- Perhaps a small explanation would be still conceivable, concerning the difference between the two cards
- Is it possible to take out the option of "renewal application" when selecting "golden honorary card"?

Requirements:
- What happens if the applicant is not yet 16 years old? Shouldn't this also be asked for in the prerequisites (for the blue honorary activity card)? For example: I have reached the age of 16 and meet the following prerequisite for applying for a blue honorary activity card: * (Otherwise we would have again problems regarding data protection, because possibly data of persons are processed, who are not yet 16).
- What do you think: Should the Bavarian Minister President's badge of honor also be gendered? So far, the case is quite clear, but who knows what the future will bring J Or could this be changed later?
- There is no * in proof of activity to indicate that this is a required field. However, a file must be attached to continue.

**Comment by maxammann** [comment]
We will need to filter these responses into the following two categories:
1. Change Requests which concern things we did not understand understand initially
2. CR which are an improvement to the app but not neccassary
3. CR which are very minor (<30s to change)

I would say we do 1. and 3. but do 2. only after the release.
E.g. "Is it possible to take out the option of "renewal application" when selecting "golden honorary card"?" is a new task and a request for a new feature. We will do that only after the release.

I would definitely do the "label adjustments".

**Comment by michael-markl** [comment]
I've looked through the requested changes and fixed up the "easy ones". (PR incoming...)

[ I implemented the requirement that we need to check that applicants are 16 or older, by checking whether the date of birth is actually at least 16 years ago. ]

What's left:

* "Nice to have: Tätig seit *: Can this date picker easiley be adjusted to month+year only instead of exact day?"
   As this is not as easy as changing a single line, I would drop this.

* After submitting add sth like "Für Rückfragen zum Vorgang notieren Sie sich bitte folgende Bearbeitungs-ID [REGIONID]-XXXX". I think this would be great, but is also not a (super) low-hanging fruit right now. I will open an issue for that.
